src/screens/login/login.js

```diff
@@ -29,20 +29,20 @@ export default class LoginScreen {
     const email = formData.get('email');
     const password = formData.get('password');
 
+    Spinner.show(target.querySelector('.login__content'));
```

Comment:
Where is hide?

Reply:
In this case hide() is useless, because login screen will call render() anyway.